tests/transactional/enable_selinux.pm

```diff
@@ -13,11 +13,32 @@ use strict;
 use warnings;
 use testapi;
 use transactional qw(process_reboot);
-use version_utils qw(is_sle_micro is_public_cloud);
+use version_utils qw(is_leap_micro is_microos is_sle_micro is_public_cloud);
 use Utils::Systemd qw(systemctl);
 use Utils::Logging 'save_and_upload_log';
 use transactional qw(trup_call check_reboot_changes process_reboot);
 
+sub check_enforcing {
+    assert_script_run('selinuxenabled');
+    validate_script_output("getenforce", sub { m/Enforcing/ });
+    validate_script_output("sestatus", sub { m/Current mode:.*enforcing/ });
+    validate_script_output("sestatus", sub { m/Mode from config file:.*enforcing/ });
+    record_info('SELinux', script_output('sestatus'));
```

Comment:
I would move those checks up, so also in case of failure we have the output present in the openQA test run for investigation purposes.